Automattic/sensei
# Update codebase style

### Changes proposed in this Pull Request

* Related to [this PR](https://github.com/Automattic/sensei/pull/3563), it reformats our whole codebase (JS and CSS), so we don't need to send reformated code every time we change an old file.

### Testing instructions

* Build the project (`npm run build`).
* Install the plugin in a WP site.
* Navigate/interact in the pages with CSS and JS, like the Setup Wizard, Importer, Exporter, Learner management, the course page in the frontend, and so on. And make sure everything is working well without any warning on the console or unexpected behavior.
